### PR TITLE
fix crosschain lag

### PR DESCRIPTION
**Pull request by nguyenhoaidanh**
- hooks useActiveWeb3React re-render too much (maybe come from web3-react). we can add console.log() in that hooks to see too nany logs
- use useChainsConfig in this hooks can cause lag: [ticket here](https://team-kyber.atlassian.net/browse/CR-656?focusedCommentId=16953&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-16953)
- So I decide to bring out NETWORKS_INFO outside useChainsConfig https://github.com/KyberNetwork/kyberswap-interface/blob/78c59818de5f80df775aa897687e3f2c7f9dd92c/src/hooks/useChainsConfig.ts

### CODE REVIEW

File: src/hooks/useChainsConfig.ts [hunk 1]
```diff
@@ -15,7 +15,16 @@ export enum ChainState {
 
 export type ChainStateMap = { [chain in ChainId]: ChainState }
 
-const defaultData = MAINNET_NETWORKS.map(chainId => NETWORKS_INFO[chainId])
+const cacheInfo: { [chain: string]: NetworkInfo } = {}
+// todo danh, when chain setting from admin ready, update all place use this
```
Comment (namgold) [inline src/hooks/useChainsConfig.ts:19]: :danh:

Comment (nguyenhoaidanh) [reply src/hooks/useChainsConfig.ts:19]: later sir

**Review by viet-nv — APPROVED** [review]
Can you brief the problem and solution of this PR?

**Comment by nguyenhoaidanh** [comment]
updated pr desc

**Pull request merged**